.gitignore

```diff
@@ -0,0 +1 @@
+.build
```

Comment:
I tend to use `build` in my own projects, but I also prefer building entirely out of tree. However, seeing `.build` it occurs to me that it will basically never collide with any meaningful artefact in the repository, so a point in its favor?